src/main/cliapp/gulpfile.js

```diff
@@ -1,17 +0,0 @@
-const gulp = require("gulp");
-const inlinesource = require("gulp-inline-source");
-const replace = require("gulp-replace");
-
-gulp.task("default", () => {
-  return gulp
-    .src("./build/*.html")
-    .pipe(replace('.js"></script>', '.js" inline></script>'))
-    .pipe(replace('rel="stylesheet">', 'rel="stylesheet" inline>'))
-    .pipe(
-      inlinesource({
-        compress: false,
-        ignore: ["png"],
-      })
-    )
-    .pipe(gulp.dest("./build"));
-});
```

Comment:
@adamgibs I do believe we are going to need this file for building the app... would need to verify.